packages/markdown-docx/src/CiceroMarkToOOXMLRules.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { sanitizeHtmlChars } = require('./CiceroMarkToOOXMLHelpers');
+
+/**
+ * Inserts text.
+ *
+ * @param {string}  value Enclosing value of the OOXML tag
+ * @returns {string} OOXML tag for the text
+ */
+const TEXT_RULE = (value) => {
+    return `<w:r><w:t xml:space="preserve">${sanitizeHtmlChars(value)}</w:t></w:r>`;
+};
+
+/**
+ * Inserts emphasised text.
+ *
+ * @param {string}  value  Enclosing value of the OOXML tag
+ * @returns {string} OOXML tag for the emphasised text
+ */
+const EMPHASIS_RULE = (value) => {
+    return `<w:r><w:rPr><w:i w:val="true" /></w:rPr><w:t>${sanitizeHtmlChars(value)}</w:t></w:r>`;
```

Comment:
`w:val="true"` is not needed like I reasoned before.

Reply:
@algomaster99 
It was present in the initial converter (cicero-add-in) so I didn't change it. Should I remove this/

Reply:
@algomaster99 
Done

Reply:
> Address [this comment](https://github.com/accordproject/markdown-transform/pull/401#discussion_r649837073) and we would be good to go.

@algomaster99 
I already made the changes for this.